### PR TITLE
Updated spanish localisation

### PR DESCRIPTION
**Pull request by n4al**

### CODE REVIEW

File: python/web/src/translations/es/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -8,8 +8,8 @@ msgid ""
 msgstr ""
 "Project-Id-Version: PiSCSI 68kmla Edition\n"
 "Report-Msgid-Bugs-To: https://github.com/akuker/PISCSI/issues\n"
```
Comment (rdmark) [inline python/web/src/translations/es/LC_MESSAGES/messages.po:10]: Minor thing, but this URL needs to be updated to https://github.com/PiSCSI/piscsi/issues

File: python/web/src/translations/es/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -1183,3 +1245,87 @@ msgstr "Registros del sistema: %(scope)s %(lines)s líneas"
...
+#~ "File Server."
+#~ msgstr ""
+#~ "Instale <a href=“%(url)s” target=“_blank”>Netatalk</a> para utilizar el "
+#~ "servidor de archivos AFP."
+
+#~ msgid "Reboot Raspberry Pi"
+#~ msgstr "Reiniciar la Raspberry Pi"
+
+#~ msgid "Shut Down Raspberry Pi"
+#~ msgstr "Apagar la Raspberry Pi"
```
Comment (rdmark) [inline python/web/src/translations/es/LC_MESSAGES/messages.po:1331]: Don't forget to delete these lines!

**Review by rdmark — APPROVED** [review]

File: python/web/src/translations/es/LC_MESSAGES/messages.po (hunk 2 above)
Comment (n4al) [reply python/web/src/translations/es/LC_MESSAGES/messages.po:1331]: If you don't mind me asking, why? And how do I know they should be deleted? They seemed as legitimate.

Comment (rdmark) [reply python/web/src/translations/es/LC_MESSAGES/messages.po:1331]: Every line that begins with `#~` is a discarded line. They are the translations of strings that were removed, or changed too drastically for gettext to match them with your existing translations.

The reason you want to delete them is to remove the clutter and save space in the repo. They are there only for you to review and reuse if needed.

**Pull request merged**